source/user-manual/registering/index.rst

```diff
@@ -8,56 +8,118 @@ Registering agents
 .. meta::
   :description: Learn more about the different methods that can be used to register agents against the Wazuh manager.
 
-In this section, the registration process is described, and more specifically the different methods that users can use to register agents in the Wazuh manager.
+Wazuh agents communicate with the Wazuh server using outbound TCP/UDP connections to two different services:
```

Comment:
It is not said that 1515 and 1514 are default ports.

Reply:
Wazuh agent communicates with the Wazuh manager

Reply:
Done